File: Java/Array/linearSearch.java
import java.util.*;

class Array{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);

        //size
        System.out.println("Enter size of array:");
        int n = sc.nextInt();
        int number[] = new int[n];

        //Elements
        System.out.println("ENter elements of array:");
        for(int i=0;i<n;i++){
            number[i] = sc.nextInt();
        }

        //elements to be searched
        System.out.println("Enter elements to be searched:");
        int x = sc.nextInt();

        for(int i=0;i<number.length;i++){
            if(number[i]==x){
            System.out.println("x is found at "+i);
            }
        }
    }
}

File: C Program/Searching Technique/LinearSearch.c
#include<stdio.h>

int main(){
    int n,flag = 0;
    printf("Enter size of an array:");
    scanf("%d",&n);

    int arr[n];
    printf("Enter array elements:");
    for(int i=0;i<n;i++){
        scanf("%d",&arr[i]);
    }

    int key;
    printf("Enter a key element to be searched:");
    scanf("%d",&key);

    for(int i=0;i<n;i++){
        if(arr[i] == key){
            flag = 1;
            break;
        }
    }
    if(flag){
        printf("Key elements found.");
    }
    else{
        printf("Not Found");
    }
    return 0;
}